cilium/api/bpf_metadata.proto

```diff
@@ -55,4 +55,13 @@ message BpfMetadata {
   // l7lb_policy_name is the name of the L7LB policy that is enforced on the listener.
   // This is optional field.
   string l7lb_policy_name = 10;
+
+  // original_source_so_linger_time specifies the number of seconds to linger on socket close.
+  // Only used if use_original_source_address is also true, and the original source address
```

Comment:
From the agent side, we probably need to set this _always_ whenever use_original_source_address is true right ?

Reply:
ah just saw this one https://github.com/cilium/cilium/pull/38500